shared/app/nav.native.js

```diff
@@ -75,7 +75,8 @@ class CardStackShim extends Component<CardStackShimProps> {
           .map((route, index) => {
             const routeName = route.path.join('/')
             // The bottom/back item of the stack is our top (active) screen
-            const shouldRender = !this.props.hidden && (index === stack.size - 1 || index === stack.size - 2)
+            const shouldRender =
```

Comment:
I have another pr with just this. since you can't swipe back on android it seems worth it to just render the topmost always